polkadot/xcm/pallet-xcm/src/benchmarking.rs

```diff
@@ -28,7 +28,7 @@ type RuntimeOrigin<T> = <T as frame_system::Config>::RuntimeOrigin;
 pub struct Pallet<T: Config>(crate::Pallet<T>);
 
 /// Trait that must be implemented by runtime to be able to benchmark pallet properly.
-pub trait Config: crate::Config {
+pub trait Config: crate::Config + pallet_balances::Config {
```

Comment:
Just thinking, will we keep `pallet_xcm` on the relaychain after AHM? Probably yes, because of `pallet_xcm::execute`? But I am not sure about `pallet_balances`? Could this `+ pallet_balances::Config` cause problems later?

Reply:
I don't think so, it is only used here in benchmarks to allow benchmark to add prior balance for Consideration charging.

Removing balances pallet from Relay is stage 3 of AHM (estimated 2026), when we get there we will probably have to create hide this behind a feature flag. I wouldn't do it yet though.

Reply:
Aha, ok, I got it.
We could also migrate to `fungible` traits: https://github.com/paritytech/polkadot-sdk/issues/7226, not sure if it makes things easier later.